### PR TITLE
[wasmedge] Update to 0.12.1

### PR DESCRIPTION
**Pull request by Pospelove**
patch has been upstreamed https://github.com/WasmEdge/WasmEdge/pull/2443

### CODE REVIEW

**Review by FrankXie05 — APPROVED** [review]

File: ports/wasmedge/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "wasmedge",
-  "version-semver": "0.12.0-alpha.1",
-  "port-version": 1,
+  "version-semver": "0.12.1",
```
Comment (FrankXie05) [inline ports/wasmedge/vcpkg.json:3]:
```suggestion
  "version": "0.12.1",
```
We don't have to use the syntax of `semver`, it only applies to `alpha`. Unless there is an explicit instruction to use `semver`.

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

**Comment by FrankXie05** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review".
That way, I can be aware that you've responded since you can't modify the tags.

**Comment by Pospelove** [comment]
@FrankXie05 ready for review

File: versions/w-/wasmedge.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "b4427ec8cddcee004d30d5b4b3fff43319251638",
+      "version-semver": "0.12.1",
+      "port-version": 0
+    },
```
Comment (FrankXie05) [inline versions/w-/wasmedge.json:7]: Need to remove and re-run command `./vcpkg x-add-version wasmedge`.

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: versions/w-/wasmedge.json (hunk 2 above)
Comment (FrankXie05) [reply versions/w-/wasmedge.json:7]: I will fix it. :)

**Review by FrankXie05 — APPROVED** [review]

**Pull request merged**